lib/vtls/schannel.c

```diff
@@ -884,14 +884,13 @@ schannel_recv(struct connectdata *conn, int sockindex,
                            size, &nread);
     /* check for received data */
     if(*err != CURLE_OK)
-      ret = -1;
+      return -1;
     else {
```

Comment:
Hope I'm not wading in where I'm not wanted.  With a return here, I think it's OK to drop the else, and potentially simplify, or at least remove a level of indentation.